docs/models.md

```diff
@@ -2,36 +2,37 @@
 
 ## Multiple Adapters
 
-It is possible to register multiple adapters with different names/URLs, for example:
+It is possible to register multiple connections, for example:
 
 ```Crystal
-Granite::Adapters << Granite::Adapter::Mysql.new({name: "legacy_db", url: "LEGACY_DB_URL"})
-Granite::Adapters << Granite::Adapter::Pg.new({name: "new_db", url: "NEW_DB_URL"})
+Granite::Connections << Granite::Adapter::Mysql.new(name: "legacy_db", url: "LEGACY_DB_URL")
+Granite::Connections << Granite::Adapter::Pg.new(name: "new_db", url: "NEW_DB_URL")
 
 class Foo < Granite::Base
-  adapter legacy_db
+  connection legacy_db
   
   # model fields
 end
 
 class Bar < Granite::Base
-  adapter new_db
+  connection "new_db"
```

Comment:
Using quotes here might be confusing or considered a typo since this section isn't related to the type provided to the macro.